content/root/best-practice-guide.textile

```diff
@@ -152,15 +173,15 @@ A client ID serves as an identification for your clients when they are present o
 
 h3(#use-existing-presence-set). Using the synced presence set provided by the realtime library
 
-Building a mechanism that keeps a tab on presence events can be quite complex. Hence, the @Presence@ object in the realtime library exposes a @get@ "method":https://ably.com/documentation/realtime/presence#get allowing a client to retrieve an array of all members present on the channel. The Ably client is responsible for keeping track of the presence set from the time that the channel is attached. An up to date presence set is pushed to the client following attach and this set is updated on each subsequent presence event. Thus get returns the already-known presence set retained in memory and does not trigger a new request to the Ably service. You can simply use this method as shown below, instead of venturing into the complexity of building one of your own.
+Building a mechanism that keeps a tab on presence events can be quite complex. For this reason, the @Presence@ object in the realtime library exposes a @get@ "method":https://ably.com/documentation/realtime/presence#get allowing a client to retrieve an array of all members present on the channel. The Ably client is responsible for keeping track of the presence set from the time that the channel is attached. An up-to-date presence set is pushed to the client following attach and this set is updated on each subsequent presence event. Thus @get@ returns the already-known presence set retained in memory and does not trigger a new request to the Ably service. You can simply use this method as shown below, instead of venturing into the complexity of building one of your own.
```

Comment:
```suggestion
Monitoring presence events can be quite complex. For this reason, the @Presence@ object in the realtime library exposes a @get@ "method":https://ably.com/documentation/realtime/presence#get allowing a client to retrieve an array of all members present on the channel. The Ably client is responsible for keeping track of the presence set from the time that the channel is attached. An up-to-date presence set is pushed to the client following attach and this set is updated on each subsequent presence event. Thus @get@ returns the already-known presence set retained in memory and does not trigger a new request to the Ably service. You can simply use this method as shown below, instead of venturing into the complexity of building one of your own.
```

I think "keep tabs on" is a colloquial phrase